apps/ejabberd/src/mod_push_service_mongoosepush.erl

```diff
@@ -44,7 +44,8 @@
 %%--------------------------------------------------------------------
 
 -spec start(Host :: ejabberd:server(), Opts :: list()) -> any().
-start(Host, Opts) ->
+start(_ServerHost, Opts) ->
+    Host = ?MYNAME,
```

Comment:
@michalwski Yes, it should. The issue was that `Host` is not available in the `node_push'a` callbacks. I must have forgotten to to change `mod_pubsub` to include this variable in the `publish` callback. I'm gonna take care of that.